src/Tizen.CoreUI/src/CoreUI.Canvas/AlphaAnimation.cs

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2019 by its authors. See AUTHORS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma warning disable CS1591
+using System;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Diagnostics.Contracts;
+namespace CoreUI.Canvas {
+    /// <summary>Animated alpha blending effect.
+    /// 
+    /// The <see cref="CoreUI.Canvas.Object"/> will have its alpha (transparency) value multiplied by this animation. This can be used to create fade-in and fade-out transitions, for example.</summary>
+    /// <since_tizen> 6 </since_tizen>
+    [CoreUI.Canvas.AlphaAnimation.NativeMethods]
+    [CoreUI.Wrapper.BindingEntity]
+    [SuppressMessage("Microsoft.Naming", "CA1724:TypeNamesShouldNotMatchNamespaces")]
+    public class AlphaAnimation : CoreUI.Canvas.Animation
+    {
+        /// <summary>Pointer to the native class description.</summary>
+        public override System.IntPtr NativeClass
```

Comment:
Why this IntPtr is public? Maybe you can wrap it into some SafeHandle or redesign it.